mentat/config.py

```diff
@@ -35,22 +35,52 @@ class Config:
     # Model specific settings
     model: str = attr.field(
         default="gpt-4-1106-preview",
-        metadata={"auto_completions": list(known_models.keys())},
+        metadata={
+            "auto_completions": list(known_models.keys()),
+            "description": "Choose from one of the supported models: " + ",".join(
+                list(known_models.keys())
+            ),
+        },
     )
     feature_selection_model: str = attr.field(
         default="gpt-4-1106-preview",
-        metadata={"auto_completions": list(known_models.keys())},
+        metadata={
+            "auto_completions": list(known_models.keys()),
+            "description": (
+                "Choose from one of the supported feature selection models: "
```

Comment:
I prefer "The model used to prune the features before sending to the main model for edits. Only used if `llm_feature_filter` is greater than 0."